### PR TITLE
check app is enabled

### PR DESCRIPTION
**Pull request by ArtificialOwl**
verify app set in route_to_circle is enabled

### CODE REVIEW

**Review by mejo- — APPROVED** [review]

File: lib/Model/ModelManager.php [hunk 1]
```diff
@@ -554,16 +540,24 @@ public function fixInstance(string $instance): string {
...
+			$path = $this->configService->getAppValue(ConfigService::ROUTE_TO_CIRCLE);
+			$pos = strpos($path, '.');
+			if (!$pos) {
+				return '';
+			}
 
-		try {
-			if ($path !== '') {
-				return $this->urlGenerator->linkToRoute($path, ['singleId' => $singleId]);
+			if (!$this->appManager->isInstalled(substr($path, 0, $pos))) {
```
Comment (juliushaertl) [inline lib/Model/ModelManager.php:551]: This should check if the app is enabled, since routes are only loaded in that case.

**Review by juliushaertl — CHANGES_REQUESTED** [review]

File: lib/Model/ModelManager.php (hunk 1 above)
Comment (ArtificialOwl) [reply lib/Model/ModelManager.php:551]: isInstalled() check if app is enabled, as said in method phpdoc :)

Comment (juliushaertl) [reply lib/Model/ModelManager.php:551]: I'd say even better reason to use the other one to be more explicit 🙈 We should actually deprecate that app manager method then.

**Review by juliushaertl — APPROVED** [review]
Not blocking but I'd prefer the other method 👍

**Pull request merged**